src/transformers/utils/quantization_config.py

```diff
@@ -215,11 +209,8 @@ def __init__(
         self,
         nbits: int = 4,
         group_size: int = 64,
-        quant_zero: bool = True,
-        quant_scale: bool = False,
-        offload_meta: bool = False,
```

Comment:
Are these deprecated in the latest version of hqq ? Would it make sense to enforce the user to download the latest version of hqq ?

Reply:
Yeah, I saw that transformers installs the latest version of hqq in the docker file

Reply:
Then, maybe we can update `is_hqq_available` to force the user to install the latest hqq.

Reply:
Like this https://github.com/huggingface/transformers/pull/33141/commits/9f7c23577b2f13423250dcb766dec0d421fe71ea ?

Reply:
yes could you also update the `validate_environment` function where this check is used to remind the user that he needs to upgrade their version ?

Reply:
>  Also, we can potentially let the user know that these args are deprecated if they set it in the latest version of hqq.

if we see these in the kwargs, we need to warn them that they are deprecated.

Reply:
`validate_environment` message mentioning at least `0.2.0`: https://github.com/huggingface/transformers/pull/33141/commits/4682a725c52ab2b2bb00e021b6e145d1f6d0abb6

Deprecated keys in kwargs message:
https://github.com/huggingface/transformers/pull/33141/commits/813ed627b3079f53e0423f0f24a3be78a30c169c

Reply:
nice !